### PR TITLE
Add external zones article

### PR DESCRIPTION
**Pull request by lokst**
Document the new external zones feature

- [x] Managing records
- [x] Supported record types
- [x] Update screenshots with latest screen elements
  - [x] Refresh button
  - [x] Sync text 
- [x] Optimize images

### CODE REVIEW

**Comment by dallasread** [comment]
This looks good to me. I think it is ready for review as soon as record refreshing is added! 🙇‍♂️

File: content/articles/external-dns-providers.markdown [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+To proceed with linking [Amazon Route 53](https://aws.amazon.com/route53/) as an external DNS provider to your account, you must:
+
+- Have an [AWS access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for an AWS user with permission to manage public hosted zones at Route 53.
+- Have administrator access to a DNSimple account
+
+## Supported record types {#supported-record-types}
+
+Records of supported record types can be [synced](/articles/deployment-editor#record-syncing) from/to DNSimple to/from an external DNS provider.
+However, record fields that are not supported in DNSimple records will not be synced. Likewise, record fields like [record notes](/articles/record-notes) and [regional record](/articles/regional-records) information are only supported in DNSimple records and will not be synced to the external DNS provider.
```
Comment (dallasread) [inline content/articles/external-dns-providers.markdown:38]:
```suggestion
However, record fields that are not supported by the destination provider will be ignored. Likewise, record fields like [record notes](/articles/record-notes) and [regional record](/articles/regional-records) information are only supported by DNSimple and will not be synced to the external DNS provider.
```

File: content/articles/deployment-editor.markdown [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+1.  Enter the required infomation and press <label>Delete Record</label> to delete the record. For each DNS provider that the record should be deleted at, ensure that the corresponding checkbox is checked. Note that the record note will only be saved if DNSimple is one of the selected providers.
+
+    ![Deployment Editor Delete Record](/files/deployment-editor-delete-record.png)
+</div>
+
+## Refreshing external zone records {#refreshing-external-zone-records}
+
+If you have made changes to your external zone records outside of DNSimple, you can refresh their state in
+DNSimple using the <label>Refresh records</label> button in the Deployment Editor.
```
Comment (dallasread) [inline content/articles/deployment-editor.markdown:97]:
```suggestion
If you have made changes to your external zone records outside of DNSimple, you can refresh their state in DNSimple using the <label>Refresh records</label> button in the Deployment Editor.
```

File: content/articles/external-dns-providers.markdown [hunk 3]
```diff
@@ -0,0 +1,69 @@
+---
+title: External DNS Providers
+excerpt: Link an external DNS provider to your DNSimple account to manage zones at other authoritative DNS providers, all within DNSimple.
+categories:
+- DNS
```
Comment (dallasread) [inline content/articles/external-dns-providers.markdown:5]: I wonder if this should also be in `Integrations`.

File: content/articles/external-dns-providers.markdown (hunk 1 above)
Comment (lokst) [reply content/articles/external-dns-providers.markdown:38]: What do you think of `record fields that are not supported by DNSimple, or the destination provider` ?

File: content/articles/external-dns-providers.markdown (hunk 3 above)
Comment (lokst) [reply content/articles/external-dns-providers.markdown:5]: I think that's a great idea! Added it: https://github.com/dnsimple/dnsimple-support/pull/1032/commits/063a34d9040a1d9f1bbc0274f3e416404e633b69

**Review by kojdm — APPROVED** [review]
This LGTM!

File: content/articles/external-dns-providers.markdown (hunk 1 above)
Comment (oinvertedworld) [reply content/articles/external-dns-providers.markdown]: @dallasread Made the change: https://github.com/dnsimple/dnsimple-support/pull/1032/files/ec05b9231e50ff9f885b25711558094ded0be737..a775c9357ef914c11a8fb1035aa2ed1dbd69710b (cc @itsalyse)

Comment (lokst) [reply content/articles/external-dns-providers.markdown]: @dallasread Made the change: https://github.com/dnsimple/dnsimple-support/pull/1032/files/ec05b9231e50ff9f885b25711558094ded0be737..a775c9357ef914c11a8fb1035aa2ed1dbd69710b (cc @itsalyse )

File: content/articles/external-dns-providers.markdown [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+### Amazon Route 53
+
+DNSimple supports [Amazon Route 53](https://aws.amazon.com/route53/) as an external DNS provider.
+
+#### Prerequisites for linking Amazon Route 53 as an external DNS provider
+
+To link [Amazon Route 53](https://aws.amazon.com/route53/) as an external DNS provider, you need:
+
+- An [AWS access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for an AWS user with permission to manage public hosted zones at Route 53.
```
Comment (dallasread) [inline content/articles/external-dns-providers.markdown:33]: Should we list the minimum permissions required? https://github.com/dnsimple/dnsimple-ops/issues/3193#issue-1710471000

Comment (lokst) [reply content/articles/external-dns-providers.markdown]: Added in https://github.com/dnsimple/dnsimple-support/pull/1032/commits/7c4857bd0a477c938380de52e06cdaec32b90d3d 🎉

File: categories/dns.yaml [hunk 5]
```diff
@@ -35,6 +35,9 @@ Zones:
 - "Domain Zone Files"
 - "DNS Hosting"
 - "Dynamic DNS"
+- "External DNS Providers"
```
Comment (aeden) [inline categories/dns.yaml:38]: Didn't we agree to change the term to Integrated DNS Providers?

Comment (lokst) [reply categories/dns.yaml:38]: Yes, we'll be updating the UI and support articles accordingly 👍

**Pull request merged**